unitybuild-start/action.yml

```
@@ -86,6 +86,21 @@ runs:
             -d"{ $label_json, $commit_json }" \
             "${{ inputs.base_url }}$path"
           )
+          if [ "${http_code}" = "502" ]; then
+            attempts=0
+            while [ "$http_code" != "202" ] || [ attempts != 3 ]
+              do
+                sleep 30
+                http_code=$(curl -s \
+                  -w'%{http_code}' \
+                  -o tmpoutput.json \
+                  -H'Content-Type: application/json' \
+                  -H'Authorization: Basic ${{ inputs.api_key }}' \
+                  -d"{ $label_json, $commit_json }" \
+                  "${{ inputs.base_url }}$path"
+                ((attempt=attempt+1))
```

Выглядит странно.

```suggestion
                attempt=$((attempt+1))
```

или

```suggestion
                let attempt=attempt+1
```